packages/create-cloudflare/templates/remix/c3.ts

```diff
@@ -10,24 +13,55 @@ const generate = async (ctx: C3Context) => {
 	await runFrameworkGenerator(ctx, [
 		ctx.project.name,
 		"--template",
-		"https://github.com/remix-run/remix/tree/main/templates/cloudflare-pages",
+		"https://github.com/remix-run/remix/tree/main/templates/vite-cloudflare",
 	]);
 
 	logRaw(""); // newline
 };
 
+const configure = async () => {
+	const typeDefsPath = "load-context.ts";
+
+	const s = spinner();
+	s.start(`Updating \`${typeDefsPath}\``);
+
+	// Remove the empty Env declaration from the template to allow the type from
+	// worker-configuration.d.ts to take over
+	transformFile(typeDefsPath, {
+		visitTSInterfaceDeclaration(n) {
+			if (n.node.id.type === "Identifier" && n.node.id.name !== "Env") {
+				return this.traverse(n);
+			}
+
+			// Removes the node
+			n.replace();
+			return false;
+		},
+	});
+
+	s.stop(`${brandColor("updated")} \`${typeDefsPath}\``);
```

Comment:
Isn't this missing the `dim`?

![Screenshot 2024-02-24 at 00 33 39](https://github.com/cloudflare/workers-sdk/assets/61631103/aceabcf3-d00b-4c65-8a88-bddf9e24e045)


```suggestion
	s.stop(`${brandColor("updated")} ${dim(`\`${typeDefsPath}\``)}`);
```